templates/python/custom-copilot-basic/README.md.tpl

```diff
@@ -0,0 +1,105 @@
+# Overview of the AI Chat Bot template
+
+This template showcases a bot app that responds to user questions like an AI assistant. This enables your users to talk with the AI assistant in Teams to find information.
+
+The app template is built using the Teams AI library, which provides the capabilities to build AI-based Teams applications.
+
+- [Overview of the AI Chat Bot template](#overview-of-the-ai-chat-bot-template)
+  - [Get started with the AI Chat Bot template](#get-started-with-the-ai-chat-bot-template)
+  - [What's included in the template](#whats-included-in-the-template)
+  - [Extend the AI Chat Bot template with more AI capabilities](#extend-the-ai-chat-bot-template-with-more-ai-capabilities)
+  - [Additional information and references](#additional-information-and-references)
+
+## Get started with the AI Chat Bot template
+
+> **Prerequisites**
+>
+> To run the AI Chat Bot template in your local dev machine, you will need:
+>
+> - [Python](https://www.python.org/), version 3.11 or higher
+> - [Python extension](https://code.visualstudio.com/docs/languages/python), version v2024.0.1 or higher
+> - [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version 5.0.0 and higher or [Teams Toolkit CLI](https://aka.ms/teamsfx-cli)
+{{#useAzureOpenAI}}
+> - An account with [Azure OpenAI](https://aka.ms/oai/access).
+{{/useAzureOpenAI}}
+{{#useOpenAI}}
+> - An account with [OpenAI](https://platform.openai.com/).
+{{/useOpenAI}}
+{{^enableTestToolByDefault}}
+> - A [Microsoft 365 account for development](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts)
+{{/enableTestToolByDefault}}
+{{#enableTestToolByDefault}}
+> - [Node.js](https://nodejs.org/) (supported versions: 16, 18) for local debug in Test Tool.
+{{/enableTestToolByDefault}}
+
+1. First, Press **CTRL+Shift+P** to open the command box and enter `Python: Create Environment` to create and activate your desired virtual environment. Remember to select `src/requirements.txt` as dependencies to install when creating the virtual environment.
```

Comment:
Different os has different shortcut. List all of them or just not mention the shortcut